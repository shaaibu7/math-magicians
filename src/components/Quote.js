import React from 'react';

const Quote = () => {
  <>
    <p>
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding.
      <span>-William Paul Thurston</span>
    </p>
  </>;
};

export default Quote;
