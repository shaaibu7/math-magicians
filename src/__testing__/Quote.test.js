import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';
import '@testing-library/jest-dom';

test('expect component to match snapshot', () => {
  const quotes = renderer.create(<Quote />).toJSON();
  expect(quotes).toMatchSnapshot();
});
