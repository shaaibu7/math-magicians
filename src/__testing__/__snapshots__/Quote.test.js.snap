// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect component to match snapshot 1`] = `
<div
  className="text"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <span>
      - William Paul Thurston
    </span>
  </p>
  <p>
    Pure mathematics is, in its way, the poetry of logical ideas.
    <span>
      - Albert Einstein
    </span>
  </p>
  <p>
    Mathematics is the most beautiful and most powerful creation of the human spirit.
    <span>
      - Stefan Banach
    </span>
  </p>
</div>
`;
